src/js/index.js

```diff
@@ -0,0 +1,91 @@
+import {navigation} from '../../Modules/navigation.js';
+import { myDate } from "../../Modules/date.js";
+
+const form = document.getElementById('form');
+const titleInput = document.getElementById('title-input');
+const authorInput = document.getElementById('author-input');
+const booksList = document.getElementById('books');
+
+let books = [];
+let id = 0;
+
+class Book {
+  constructor(title, author, id) {
+    this.title = title;
+    this.author = author;
+    this.id = id;
+  }
+}
+
+class UI {
+  static loadBooks() {
+    const storedBooks = JSON.parse(localStorage.getItem('books'));
+    if (storedBooks) {
+      books = storedBooks;
+      books.forEach(book => UI.createBook(book));
+    }
+  }
+
+  static createBook(book) {
+    const newBook = document.createElement('tr');
+    newBook.innerHTML = `
+      <td>"${book.title}" by ${book.author}
+      <button type="button" class="remove-btn" data-id="${book.id}">Remove</button>
+      </td>
+    `;
+    booksList.appendChild(newBook);
+  }
+
+  static clearFields() {
+    titleInput.value = '';
+    authorInput.value = '';
+  }
+
+  static removeBook(id) {
+    const index = books.findIndex(book => book.id === id);
+    if (index !== -1) {
+      books.splice(index, 1);
+      booksList.innerHTML = '';
+      books.forEach(book => UI.createBook(book));
+      addBook();
+    }
+  }
+}
```

Comment:
- [ ] Part of the guidelines is that we divide the code into reasonable modules. Putting that into account, I would suggest you move this class definition containing the functionalities to another file under the modules folder. This `index.js` file should mostly be for importing classes and methods and running the app.

Reply:
@DammyShittu, I tried every possible way to put that class in another folder but it has a lot of variables in the static functions that stay undeclared and end up throwing errors